### PR TITLE
Annotate implementations of ILabelSymbol

### PR DESCRIPTION
**Pull request by sharwell**

### CODE REVIEW

**Comment by sharwell** [comment]
@dotnet/roslyn-compiler  for review

File: src/Compilers/CSharp/Portable/Symbols/LabelSymbol.cs [hunk 1]
```diff
@@ -1,7 +1,10 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#nullable enable
+
 using System;
 using System.Collections.Immutable;
+using System.Diagnostics.CodeAnalysis;
```
Comment (jcouv) [inline src/Compilers/CSharp/Portable/Symbols/LabelSymbol.cs]: nit: `using` could be reverted

**Comment by jcouv** [comment]
Tagging @333fred for a second review (small annotation PR)

**Comment by 333fred** [comment]
>         internal override TResult Accept<TArgument, TResult>(CSharpSymbolVisitor<TArgument, TResult> visitor, TArgument argument)

Should be annotated with `MaybeNull` on the return.

---
Refers to: src/Compilers/CSharp/Portable/Symbols/LabelSymbol.cs:123 in 05b44b8. [](commit_id = 05b44b876f68a23e1e6902f1b771b241fd416606, deletion_comment = False)

**Comment by 333fred** [comment]
>         public override TResult Accept<TResult>(CSharpSymbolVisitor<TResult> visitor)

Should be annotated with `MaybeNull` on the return.

---
Refers to: src/Compilers/CSharp/Portable/Symbols/LabelSymbol.cs:133 in 05b44b8. [](commit_id = 05b44b876f68a23e1e6902f1b771b241fd416606, deletion_comment = False)

**Comment by 333fred** [comment]
Done review pass (commit 3)

**Comment by sharwell** [comment]
> Should be annotated with MaybeNull on the return.

The C# symbol visitor is not yet annotated. These will be updated at that time.

**Comment by jcouv** [comment]
@sharwell Looks ready to merge

**Comment by sharwell** [comment]
@jcouv I'll remove the unnecessary using directive and allow a build to complete

File: src/Compilers/CSharp/Portable/Symbols/LabelSymbol.cs (hunk 1 above)
Comment (sharwell) [reply src/Compilers/CSharp/Portable/Symbols/LabelSymbol.cs]: Done!

File: src/Compilers/CSharp/Portable/Symbols/Source/SourceLabelSymbol.cs [hunk 2]
```diff
@@ -55,7 +57,7 @@ private string MakeLabelName()
             var tk = _identifierNodeOrToken.AsToken();
             if (tk.Kind() != SyntaxKind.None)
             {
-                return tk.ValueText;
+                return tk.ValueText ?? "";
```
Comment (sharwell) [inline src/Compilers/CSharp/Portable/Symbols/Source/SourceLabelSymbol.cs]: 📝 This line is impacted by #41313

**Comment by jcouv** [comment]
@sharwell Looks ready to merge

**Pull request merged**